python3.13/hello-durable/{{cookiecutter.project_name}}/tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
 pytest
 boto3
 requests
-aws-durable-execution-sdk-python-testing==1.0.0
+aws-durable-execution-sdk-python-testing=~1.0
```

Comment:
It's `~=1.0`, not `=~1.0`

Reply:
This makes me think that the tests are not checking the durable apps and we probably need to add a test on the `unit_test_pythonx_x` file: https://github.com/aws/aws-sam-cli-app-templates/blob/master/tests/integration/unit_test/test_unit_test_python3_9.py#L49